src/pytorch_lightning/utilities/types.py

```diff
@@ -27,6 +27,11 @@
 from torchmetrics import Metric
 from typing_extensions import Protocol, runtime_checkable
 
+try:
+    from torch.optim.lr_scheduler import LRScheduler as TorchLRScheduler
+except ImportError:
+    from torch.optim.lr_scheduler import _LRScheduler as TorchLRScheduler
```

Comment:
Let's maybe add a comment here with version information? Eventually, one day Lightning will have a minimum torch version requirement such that that the old import always fails.

Something along the lines of
```
# For torch<x.y.z
or
# TODO: Remove once minimum torch version is x.y.z
```

Reply:
done